src/main/checkstyle.xml

```diff
@@ -0,0 +1,250 @@
+<?xml version="1.0"?>
```

Comment:
Seems like we was need to use dependency, not file.
"Add the checkstyle maven plugin. You can use the configuration from your previous projects."